iceoryx_hoofs/source/posix_wrapper/access_control.cpp

```diff
@@ -30,15 +31,15 @@ bool AccessController::writePermissionsToFile(const int32_t fileDescriptor) cons
 {
     if (m_permissions.empty())
     {
-        std::cerr << "Error: No ACL entries defined." << std::endl;
+        IOX_LOG(ERROR) << "Error: No ACL entries defined.";
```

Comment:
With the same reasoning as in `convert.inl` I would make this `DEBUG`. The user is informed via the return value and can take some actions but would not be able to prevent a error log message even when appropriate actions are taken.